modules/ROOT/pages/delete-api-task.adoc

```diff
@@ -1,12 +1,22 @@
-= Removing an API instance on API Manager
+= Remove API Instances
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-The procedure removes the capability to manage an API Version from API Manager. If the API resides in Exchange or on the file system as a ZIP, it isn't deleted from Exchange or the file system.
+You can remove an API instance or a version of an API instance if you no longer need to manage it from API Manager. When you remove the API version from API Manager, the API is not deleted from Anypoint Exchange (Exchange) or from the file system, if it exists in any of these locations. 
 
-. In Anypoint Platform, click API Manager.
-. In API Administration, click the API version link or an instance link.
-. Select Delete from the Actions dropdown.
+Furthermore, the API asset in Exchange that the API references is not deleted.
+
+[Note] 
+--
+You cannot delete an API specification that is currently used in one or more API instances or versions being managed in API Manager. 
+--
+
+To remove API instances or API versions from API Manager:
```

Comment:
This is different from what was said above, which paired instances and version instances. 
Clarify: are these steps intended to remove one instance or a whole version and all of its instances?
Could we simple say:
```suggestion
To remove an API from API Manager:
```

Reply:
Done! Thanks!